docs/reference/configuration.md

```
@@ -29,6 +29,15 @@ createAgentPlugin({
 
 Environment variable references (`$VAR` or `${VAR}`) are resolved at startup.
 
+### Runtime identity for Apps
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)